src/test/java/step5/domain/Step5CarTest.java

```diff
@@ -0,0 +1,74 @@
+package step5.domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import step5.domain.car.Car;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+ class Step5CarTest {
+
+    @Test
+    @DisplayName("Car 생성 테스트")
+    void car_make_test() {
+        
+        String name = "홍길동";
+
+        
+        Car car = Car.defaultOf(name);
+
+        
+        assertThat(car).extracting("name").isNotNull();
+
+    }
+    
+    @Test
+    @DisplayName("기본 거리 1 테스트")
+    void default_distance_one() throws Exception {
+        String name = "홍길동";
+        Car car = Car.defaultOf(name);
+
+        boolean result = car.isDistance(1);
+
+        assertThat(result).isTrue();
+    }
+
+    @Test
+    @DisplayName("자동차 이름 5자 제한 기능 테스트")
+    void name_limit_length() {
+        
+        String name = "이름이5자이상";
+
+        
+        assertThrowsExactly(IllegalArgumentException.class, () -> Car.defaultOf(name));
+    }
+
+    @Test
+    @DisplayName("자동차 이동 테스트")
+    public void car_move_test() {
+        String name = "홍길동";
+        Car car = Car.defaultOf(name);
+
+        car.move(true);
+
+        Assertions.assertThat(car.isDistance(2)).isTrue();
```

Comment:
```suggestion
        Assertions.assertThat(car.sameDistance(2)).isTrue();
```

같냐라고 물어보는게 개인적으로는 자연스럽네요🤔